packages/contracts-bedrock/justfile

```diff
@@ -45,6 +45,18 @@ clean:
 test *ARGS: build-go-ffi
   forge test {{ARGS}}
 
+# Starts a local anvil node with a mainnet fork and sends it to the background
+anvil-fork:
```

Comment:
nit: `ETH_RPC_URL` is much more standard, I don't think we should add non standard names, also why is this coupled to mainnet? It should be able to fork any network

Reply:
It would be nice to support optionally passing thru a block number to fork from

Reply:
Will use ETH_RPC_URL. 

Created an [issue](https://github.com/ethereum-optimism/optimism/issues/13339) for the block number option.